### PR TITLE
fix: prevent name clashes when creating assets

### PR DESCRIPTION
**Pull request by wejendorp**

### CODE REVIEW

File: src/renderer/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { kapetaLight } from './Theme';
 import { initialise } from './context';
 import { PlanView } from './views/PlanView';
 import { PlanOverview } from './components/plan-overview/PlanOverview';
-import { usePlans } from './hooks/assetHooks';
+import { useLocalAssets, usePlans } from './hooks/assetHooks';
```
Comment (andertun) [inline src/renderer/index.tsx:14]:
```suggestion
import { usePlans } from './hooks/assetHooks';
```

File: src/renderer/components/forms/PlanForm.tsx [hunk 2]
```diff
@@ -16,13 +20,20 @@ interface Props extends CreatingFormProps {
...
+        [isUniqueLocally, isUniqueInRegistry]
+    );
 
     return (
         <>
             <AssetNameInput
                 name="metadata.name"
                 label="Name"
-                validation={['required']}
+                validation={validations} // , isUniqueLocally, isUniqueInRegistry]}
```
Comment (andertun) [inline src/renderer/components/forms/PlanForm.tsx:36]:
```suggestion
                validation={validations}
```

File: src/renderer/components/forms/PlanForm.tsx [hunk 3]
```diff
@@ -8,6 +8,10 @@ import React from 'react';
 import { useKapetaContext } from '../../hooks/contextHook';
 import { useNamespacesForField } from '../../hooks/useNamespacesForField';
 import { CreatingFormProps } from '../creators/AssetCreator';
+import { useValidateNameIsUniqueLocally } from '../../hooks/validationHooks';
+import { useValidateNameIsUniqueInRegistry } from '../../hooks/validationHooks';
+import { useCallback } from 'react';
+import { useMemo } from 'react';
```
Comment (andertun) [inline src/renderer/components/forms/PlanForm.tsx:14]: Nit - I noticed also in another PR that your IDE doesn't group imports.

**Review by andertun — APPROVED** [review]

File: src/renderer/components/forms/PlanForm.tsx (hunk 3 above)
Comment (wejendorp) [reply src/renderer/components/forms/PlanForm.tsx:14]: Huh, interesting. Fixing

**Pull request merged**